### DISCUSSION TITLE
File is not a database

### DISCUSSION THREAD

**Discussion by hoanm1206** [pawelsalawa/sqlitestudio#4758] [Q&A]
I expose my database and import again but no success:

It show error : file is not a database

I realized that the generated file is dump file, it is not a sql db

**Comment by pawelsalawa** [comment]
If it's a dump (i.e. file with SQL statements), then you can restore such database by creating empty, new database and right-click on it and pick "Execute SQL from file".

**Comment by hoanm1206** [comment]
@pawelsalawa 
It is not happen before, do I have to config something new?
And it should be a database instead of a dump file so I can use it right away!
